dotcom-rendering/src/model/article-schema.json

```diff
@@ -610,6 +613,7 @@
         "shouldHideAds",
         "shouldHideReaderRevenue",
         "showBottomSocialButtons",
+        "showTableOfContents",
```

Comment:
I think this might be breaking the tests, because the frontend change that will default this property to `false` isn't merged yet and the `validate.test.ts` file seems to be fetching actual data